### ISSUE TITLE
[a11y]: [Windows High Contrast Mode]: [OverflowMenu] The icon is not visible in high contrast mode with g10 theme

### ISSUE THREAD

**Issue by xrissot-ibm** [carbon-design-system/carbon#18830]
### Package

@carbon/react

### Browser

Chrome

### Package version

v1.78

### Description

OverflowMenu icon is not visible when activating Windows High Contrast Mode with Gray10 theme

### Reproduction/example

use carbon-react playground

### Steps to reproduce

Open OverflowMenu playground https://react.carbondesignsystem.com/?path=/story/components-overflowmenu--default&globals=theme:g10
Activate Windows HCM, the icon is not visible.

![Image](https://github.com/user-attachments/assets/30f2ab4d-4569-4d7f-a6d1-8961ff51c51a)

### Suggested Severity

None

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [ ] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

**Comment by heloiselui** [comment]
This was fixed in PR https://github.com/carbon-design-system/carbon/pull/19352
